tests/tests_integration/test_api/test_labels.py

```diff
@@ -25,6 +25,12 @@ def test_list(self, cognite_client, new_label, post_spy):
         assert 0 < len(res) <= 100
         assert 1 == cognite_client.labels._post.call_count
 
+    def test_retrieve(self, cognite_client):
+        res = cognite_client.labels.list(limit=1)
+        assert 1 == len(res)
+        res_retrieve = cognite_client.labels.retrieve(res[0].external_id)
+        assert res_retrieve.external_id == res[0].external_id
```

Comment:
Let's also add a check to verify that list-vs-single return works as expected:
```suggestion
        xids = res.as_external_ids()
        res_lst = cognite_client.labels.retrieve(xids)
        assert isinstance(res_lst, LabelDefinitionList)
        assert res_lst.as_external_ids() == xids

        res_single = cognite_client.labels.retrieve(xids[0])
        assert isinstance(res_single, LabelDefinition)
        assert res_single.external_id == xids[0]
```